docs/system-calls.md

```diff
@@ -0,0 +1,325 @@
+## Syscall numbers
+| Number | Name                  |
+| ------ | --------------------- |
+| 0      | [`read`](#read)       |
+| 1      | [`write`](#write)     |
+| 2      | [`open`](#open)       |
+| 3      | [`close`](#close)     |
+| 4      | [`dstat`](#dstat)     |
+| 5      | [`stat`](#stat)       |
+| 6      | [`seek`](#seek)       |
+| 7      | [`ioctl`](#ioctl)     |
+| 8      | [`mkdir`](#mkdir)     |
+| 9      | [`chdir`](#chdir)     |
+| 10     | [`curdir`](#curdir)   |
+| 11     | [`opendir`](#opendir) |
+| 12     | [`readdir`](#readdir) |
+| 13     | [`rm`](#rm)           |
+| 14     | [`mount`](#mount)     |
+| 15     | [`exit`](#exit)       |
+| 16     | [`exec`](#exec)       |
+| 17     | [`dup`](#dup)         |
+| 18     | [`msleep`](#msleep)   |
+| 19     | [`settime`](#settime) |
+| 20     | [`gettime`](#gettime) |
+| 21     | [`setdate`](#setdate) |
+| 22     | [`getdate`](#getdate) |
+| 23     | [`map`](#map)         |
+| 24     | [`swap`](#swap)       |
+
+## `read`
+Read a given descriptor.
+
+Clobbers `A`, `BC`
+
+### Parameters
+- `H` - The descriptor to read from
+- `DE` - The buffer to read into
+- `BC` - The number of bytes to read
+
+### Returns
+- `A` - 0 if successful, otherwise an error code
+- `BC` - The number of bytes read
+
+## `write`
```

Comment:
Is there a way to separate the syscalls in a nice way? Can we integrate raw HTML code?
Does `mkdocs` allow us to make custom class for a markdown block?

Reply:
There's a lot of customization you can do. IIRC there are detail HTML tags and horizontal bars in markdown, and Material for MkDocs provide a lot of [their own stuff](https://squidfunk.github.io/mkdocs-material/reference/). Not quite sure which one to use, though.